src/types/bribersDashboard.trpc.ts

```diff
@@ -8,6 +8,7 @@ export const DashboardData = z.object({
   roundEmissionsUsd: z.number(),
   totalVoteIncentives: z.number(),
   voteIncentivesRoi: z.number(),
+  externalVoteIncentivesRoi: z.number(),
```

Comment:
please make it '.optional()' to avoid conflicts with old data, that doesn't have this data